cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -1,14 +1,19 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE StrictData #-}
 
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
+
```

Comment:
:scream:

Reply:
Yeah, quite, because the API errors are too big!